pkg/operator2/starter.go

```diff
@@ -112,14 +112,7 @@ func RunOperator(ctx *controllercmd.ControllerContext) error {
 		v1helpers.EnsureOperatorConfigExists(dynamicClient, []byte(resource), gvr)
 	}
 
-	resourceSyncerInformers := map[string]informers.SharedInformerFactory{
-		targetName: informers.NewSharedInformerFactoryWithOptions(kubeClient, resync,
-			informers.WithNamespace(targetName),
-		),
-		userConfigNamespace: informers.NewSharedInformerFactoryWithOptions(kubeClient, resync,
-			informers.WithNamespace(userConfigNamespace),
-		),
-	}
+	resourceSyncerInformers := v1helpers.NewKubeInformersForNamespaces(kubeClient, targetName, userConfigNamespace)
```

Comment:
Proposal to discussion (irrelevant to this PR): should we extend this with some kind of `.WithResyncPeriod()` method?